FaunaDB.Client/Types/RefParser.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace FaunaDB.Types
+{
+    public static class RefParser
+    {
+        public static RefV Parse(Value value)
+        {
+            return value.GetOption(Field.At("@ref")).Match(
```

Comment:
I thought from the strategy document that all refs at version 2.1 would be objects, so you could just parse its properties. Did I miss anything?